umm-lib/resources/schema/echo10/Granule.xsd

```diff
@@ -353,27 +353,29 @@ xmlns:xs="http://www.w3.org/2001/XMLSchema">
       characteristics associated with a granule.</xs:documentation>
     </xs:annotation>
     <xs:sequence>
-      <xs:element minOccurs="0" name="DataGranuleSizeInBytes"
-      type="xs:unsignedLong">
+      <xs:element minOccurs="0" name="DataGranuleSizeInBytes" type="xs:unsignedLong">
         <xs:annotation>
-          <xs:documentation>The attribute will indicate the
-          volume of data in bytes contained in the
-          granule.</xs:documentation>
+          <xs:documentation>The size in Bytes of the volume of data contained in the granule.
+          Bytes are defined as eight bits. Please use this element instead of or inclusive 
+          with the SizeMBDataGranule element. The issue with SizeMBDataGranule is that end
+          users don't know if the size was calculated by using 1000xE2 Bytes (MegaBytes) or
+          1024xE2 Bytes (mebibytes) and therefore there is no systematic way to know the 
```

Comment:
I had to google mebibytes, I don't like the word. Can it be followed by '(MiB)' which I do know? Also the b in Megabytes is lowercase.